index.html

```diff
@@ -787,28 +790,42 @@
         <h2>
           <dfn>shippingAddress</dfn> attribute
         </h2>
-        <p>
-          <a>shippingAddress</a> is populated when the user provides a shipping
-          address. It is null by default. When a user provides a shipping
-          address, the <a>shipping address changed algorithm</a> runs.
+        <p class="note">
```

Comment:
@adrianhopebailie I filed: https://github.com/w3c/browser-payment-api/issues/442 

Kindly please also note you can "react" to a comment via github's interface:

![screenshot 2017-02-27 18 54 16](https://cloud.githubusercontent.com/assets/870154/23352978/3e016c6e-fd1e-11e6-97a0-2a6664eb6929.png)

It saves generating an email with only "+1" in it, while allowing you to throw support behind a proposal.